src/js/plugins/plugin.mobile_nav.js

```diff
@@ -102,6 +102,17 @@ BookReader.prototype.initToolbar = (function (super_) {
           $mmenuEl.data('mmenu').open();
         }
       });
+
+      window.addEventListener('resize', (e) => {
```

Comment:
With window resize events, it's usually a good idea to throttle how often the callback is fired to avoid an impact on performance. I'm not sure if there's already a function to do so, but the function from Petabox is pretty standard and light.